sys/slist/Makefile

```diff
@@ -0,0 +1 @@
+include $(RIOTBASE)/Makefile.base
```

Comment:
moving the slist implementation into its own PR and then rebase the olsr branch onto the slist branch would reduce the diff a little bit..